HOWTORELEASE.rst

```diff
@@ -3,11 +3,9 @@ How to release pytest
 
 Note: this assumes you have already registered on pypi.
 
-1. Bump version numbers in ``_pytest/__init__.py`` (``setup.py`` reads it).
+1. Check and finalize ``CHANGELOG.rst``.
```

Comment:
We probably want to mention to generate the tag locally right before creating the package, correct?